packages/airview-compliance-ui/src/features/application-tile/application-tile-content.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useCallback } from "react";
+import PropTypes from "prop-types";
+import IconButton from "@mui/material/IconButton";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import ExpandLessIcon from "@mui/icons-material/ExpandLess";
+import Collapse from "@mui/material/Collapse";
+import Box from "@mui/material/Box";
+
+export function ApplicationTileContent({
+  children,
+  collapsible = false,
+  initialCollapsed = false,
+  classNames,
+}) {
+  const classes = useApplicationTileContentStyles();
+
+  const [collapsed, setCollapsed] = useState(initialCollapsed);
+
+  const handleOnToggleCollapsed = useCallback(() => {
+    setCollapsed(!collapsed);
+  }, [collapsed]);
+
+  return (
+    <Box sx={classes.root} className={classNames}>
+      {collapsible && (
+        <IconButton
+          aria-label={collapsed ? "Expand content" : "Collapse content"}
+          color="primary"
+          size="small"
+          sx={{ "&.MuiIconButton-root": classes.collapseToggleBtn }}
+          onClick={handleOnToggleCollapsed}
+        >
+          {collapsed ? <ExpandMoreIcon /> : <ExpandLessIcon />}
+        </IconButton>
+      )}
+      <Collapse
+        in={!collapsed}
+        sx={classes.collapseWrapperInner}
+        aria-hidden={collapsed}
+      >
+        {children}
+      </Collapse>
+    </Box>
+  );
+}
+
+ApplicationTileContent.propTypes = {
+  /**
+   * Supported sub-components to create the ApplicationTile composition
+   */
+  children: PropTypes.node,
+  /**
+   * Sets the content to enable collapsible content toggling
+   */
+  collapsible: PropTypes.bool,
+  /**
+   * Sets the initial collapsible state (if collapsible prop is true)
+   */
+  initialCollapsed: PropTypes.bool,
+  /**
+   * Allows the passing of additional style classes to the component root node
+   */
+  classNames: PropTypes.string,
+};
+
+function useApplicationTileContentStyles() {
+  return {
+    root: {
+      padding: 2,
+    },
+    collapseToggleBtn: {
+      display: "block",
+      margin: "0 auto",
+      padding: 0,
+    },
+    collapseWrapperInner: {
+      paddingTop: 0,
```

Comment:
This is missing logic to added padding, see [here](https://github.com/AirWalk-Digital/airview-frontend/blob/70abcb226609eee1e89c660351aa2fa964eccf88/src/components/application-tile/application-tile-content.js#L77)